aiida/cmdline/commands/cmd_calcjob.py

```diff
@@ -267,8 +276,8 @@ def calcjob_cleanworkdir(calcjobs, past_days, older_than, computers, force):
         transport = orm.AuthInfo.objects.get(dbcomputer_id=computer.id, aiidauser_id=user.id).get_transport()
 
         with transport:
-            for path in paths:
-                clean_remote(transport, path)
+            for remote_folder in paths:
```

Comment:
If you fix `get_calcjob_remote_paths` to return the `Computer` instance, on line 275 you no longer need to load the `Computer` again, and you can simply get the transport with `with computer.get_transport(user):`